phpunit.integration.xml

```diff
@@ -5,9 +5,10 @@
          beStrictAboutTodoAnnotatedTests="true"
          verbose="true">
     <php>
+        <env name="setupFactory" value="\Ibexa\Contracts\Core\Test\Repository\SetupFactory\Legacy" />
         <env name="KERNEL_CLASS" value="Ibexa\Tests\Integration\Rest\IbexaTestKernel" />
         <env name="SEARCH_ENGINE" value="legacy" />
-        <env name="DATABASE_URL" value="sqlite://i@i/var/test.db" />
+        <env name="DATABASE_URL" value="sqlite://i@i/test.db" />
```

Comment:
You probably want to still put `test.db` in `var` directory. This only applies to sqlite defaults, too.
```suggestion
        <env name="DATABASE_URL" value="sqlite://i@i/var/test.db" />
```